CHANGELOG.md

```diff
@@ -0,0 +1,102 @@
+# FinOps Open Cost and Usage Specification Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+**Added:**
+
+- `RegionId` column
+
+**Changed:**
+
+- `Column naming convention` attribute includes the following new provisions:
+  - Column IDs should not exceed 50 characters.
+- `EffectiveCost` column updated to clarify it does not mix or "blend" costs across multiple charges.
+- `Region` column renamed to `RegionName`.
+- `Tags` column updated to include:
+  - Tag keys that cannot have a value must use a boolean `true` as the tag value.
+
+**Fixed:**
```

Comment:
FYI - I intentionally did not include all text changes here. I only included things that changed the implementation details. If anyone wants the lower-level text changes, they can get it from the "all changes" link in each section.